src/auto_pickup.cpp

```diff
@@ -652,85 +565,9 @@ void save_reset_changes(bool bReset)
     merge_vector();
 }
 
-std::string auto_pickup_header(bool bCharacter)
-{
-    std::string sTemp = (bCharacter) ? "character" : "global";
-    return "# This is the " + sTemp + " auto pickup rules file. The format is\n\
-# <pickup rule>;<dis/enabled>;<in/exclude>\n\n\
-# <pickup rule> Simple text. No other special characters except spaces and *\n\
-# <dis/enabled> Can be T(rue) of F(alse)\n\
-# <in/exclude> Can be T(rue) of F(alse)\n\
-#\n\
-# If # is at the start of a line, it is considered a comment and is ignored.\n\
-# In-line commenting is not allowed. I think.\n\
-#\n\
-# If you want to restore the default auto pickup rules file, simply delete this file.\n\
-# A new auto_pickup.txt will be created next time you play.\
-\n\n";
-}
-
-bool save_auto_pickup(bool bCharacter)
-{
-    std::ofstream fout;
-    std::string sFile = FILENAMES["autopickup"];
-
-    if (bCharacter) {
-        sFile = world_generator->active_world->world_path + "/" + base64_encode(g->u.name) + ".apu.txt";
-        std::ifstream fin;
-
-        fin.open((world_generator->active_world->world_path + "/" +
-                  base64_encode(g->u.name) + ".sav").c_str());
-        if(!fin.is_open()) {
-            return true;
-        }
-        fin.close();
-    }
-
-    fout.exceptions(std::ios::badbit | std::ios::failbit);
-    try {
-        assure_dir_exist(FILENAMES["config_dir"]);
-        fout.open(sFile.c_str());
-
-        fout << auto_pickup_header(bCharacter) << std::endl;
-        for( auto &elem : vAutoPickupRules[( bCharacter ) ? APU_CHARACTER : APU_GLOBAL] ) {
-            fout << elem.sRule << ";";
-            fout << ( elem.bActive ? "T" : "F" ) << ";";
-            fout << ( elem.bExclude ? "T" : "F" );
-            fout << "\n";
-        }
-
-        if (!bCharacter) {
-            merge_vector();
-            createPickupRules();
-        }
```

Comment:
As mentioned, check out #13832 for the fix; I'd appreciate at least one more pair of eyes from someone who's seen the depths of auto-pickup better than I have.